tools/devtool

```diff
@@ -660,15 +660,6 @@ apply_performance_tweaks() {
   # their maximum safe frequency. It seems to be the default for Amazon Linux, but it doesn't hurt to make this explicit.
   # See also https://wiki.archlinux.org/title/CPU_frequency_scaling
   echo performance | sudo tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor &> /dev/null
```

Comment:
should probably also update the comment on line 637

Reply:
ah, yeah, good point. Fixed!